scout/commands/load/user.py

```diff
@@ -45,10 +43,4 @@ def user(institute_id, user_name, user_mail, admin, user_id=None):
         id=user_id,
     )
 
-    user_obj = build_user(user_info)
```

Comment:
Moved this code to scout/load/user so it can be used from the cli command and from the view